pkg/deploy/generator/resources.go

```diff
@@ -1148,7 +1148,7 @@ func (g *generator) database(databaseName string, addDependsOn bool) []*arm.Reso
 						ID: to.StringPtr("[" + databaseName + "]"),
 					},
 					Options: map[string]*string{
-						"x-ms-offer-throughput": to.StringPtr("400"),
+						"throughput": to.StringPtr("500"),
```

Comment:
What's the reason for changing 400->500 here?

Reply:
I got this error from ARM when deploying my own database in step 11 following this document: https://github.com/Azure/ARO-RP/blob/master/docs/deploy-development-rp.md

"message": "Message: {"code":"BadRequest","message":"Message: {\"Errors\":[\"The offer should have valid throughput values between 500 and 1000000 inclusive in increments of 100.\"]}

Reply:
@Zhuoli interesting - I don't think we've seen that.  Please reopen changing 400 to 500 and we'll merge.

Reply:
I saw this error today updating my instance.  @Zhuoli did you want to reopen this to make the change?

Reply:
I will reopen this later today.

Reply:
@troy0820  I'm not familiar with Github PR, don't see an option to reopen this, so created another PR: https://github.com/Azure/ARO-RP/pull/299
Please advise if this is not a correct way. Thanks!